doc/source/rllib-algorithms.rst

```diff
@@ -1,6 +1,28 @@
 RLlib Algorithms
 ================
 
+Feature Compatibility Matrix
```

Comment:
I actually just moved this in https://github.com/ray-project/ray/pull/7503/files

Can you rebase the PR?